docs/tutorials/mobile/superres.md

```diff
@@ -0,0 +1,606 @@
+
+           // Step 2: get the shape of the byte array and make ort tensor
+           val shape = longArrayOf(rawImageBytes.size.toLong())
+
+           inputTensor.use {
+               // Step 3: call ort inferenceSession run
+               val output = ortSession.run(Collections.singletonMap("image", inputTensor))
+
+               // Step 4: output analysis
+               output.use {
+                   val rawOutput = (output?.get(0)?.value) as ByteArray
+                   val outputImageBitmap =
+                       byteArrayToBitmap(rawOutput)
+
+                   // Step 5: set output result
+                   result.outputBitmap = outputImageBitmap
+               }
+           }
+           return result
+       }
+
+       private fun byteArrayToBitmap(data: ByteArray): Bitmap {
+           return BitmapFactory.decodeByteArray(data, 0, data.size)
+       }
+   }
+   ```
+
+### Build and run the app
+
+Within Android studio:
+
+* Select Build -> Make Project
+* Run -> app
+
+The app runs in the device emulator. Connect to your Android device to run the app on device.
+
+## iOS app
+
+### Pre-requisites
+
+* Install Xcode 13.0 and above (preferably latest version)
+* A valid Apple Developer ID
+* An iOS device or iOS simulator
+* Xcode command line tools `xcode-select --install`
+* CocoaPods `sudo gem install cocoapods`
+
+### Sample code
+
+You can find full [source code for the iOS super resolution app](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples/super_resolution/ios) in GitHub.
+
+To run the app from source code:
+
+1. Clone the onnxruntime-inference-examples repo
+
+   ```bash
+   git clone https://github.com/microsoft/onnxruntime-inference-examples
+   cd onnxruntime-inference-examples/mobile/examples/super_resolution/ios
+   ```
+
+2. Install required pod files
+
+   ```bash
+   pod install
+   ```
+
+3. Open the generated `ORTSuperResolution.xcworkspace` file in XCode
+
+   Select your development team
+
+4. Run the application
+
+   Connect your iOS device or simulator, build and run the app
+
+   Click the `Perform Super Resolution` button to see the app in action
+
+To develop the app, step by step, follow the following sections.
+
+### Code from scratch
+
+#### Create project
+
+Create a new project in XCode
+
+#### Dependencies
+
+Install the following pods:
+
+```bash
+  # Pods for OrtSuperResolution
+  pod 'onnxruntime-c'
+  
+  # Pre-release version pods
+  pod 'onnxruntime-extensions-c', '0.5.0-dev+261962.e3663fb'
+```
+
+#### Project resources
+
+1. Add the model file to the project
+
+   Copy the model file generated at the beginning of this tutorial into the root of the project folder.
+
+2. Add the test image as an asset
+
+   Copy the image that you want to run super resolution on into the root of the project folder.
+
+#### Main app
+
+Create a file called `ORTSuperResolutionApp.swift` and add the following code:
```

Comment:
nit: this file is probably autogenerated by Xcode after creating the project

Reply:
Fixed